.ci.yaml

```diff
@@ -4105,9 +4102,10 @@ targets:
       task_name: hello_world_android__compile
 
   # mac motog4 benchmark
-  - name: Mac_android hot_mode_dev_cycle__benchmark
+  - name: Mac_arm64_android hot_mode_dev_cycle__benchmark
```

Comment:
For this kind of benchmark ideally it would be pinned to a particular model of hardware so it doesn't fluctuate between faster and slower hosts.  I believe that is the intention of hostonly `mac_benchmark`, for example.
https://github.com/flutter/flutter/blob/91f0878fedb61342535312ef35e71d04525e19c6/.ci.yaml#L125-L136

Do we need `Mac_android_benchmark` and `Mac_ios_benchmark`?

Reply:
mac_arm64_android is pinned to `device_type: "msm8952"`.

From the document no tests assigned to mac_benchmark needed to be changed.

Reply:
> mac_arm64_android is pinned to `device_type: "msm8952"`.

I mean the host Mac, as well as the device, should be pinned for this one.  This is testing regressions in the time it takes a Mac to hot reload to an Android phone, and that time would be impacted by running on a fast Mac vs a slow one.

> From the document no tests assigned to mac_benchmark needed to be changed.

I was using that as an example of where we are doing hostonly benchmarks, and therefore pin to a `mac_model`.  I think we need something similar for this test.

Reply:
Created mac_android_benchmark and moved hot_mode_dev_cycle__benchmark to it. Are there any specific tests that you would like to move to mac_ios_benchmark?

Reply:
This is getting confusing and related to https://github.com/flutter/flutter/issues/142070.

> Do we need Mac_android_benchmark and Mac_ios_benchmark?

Rethinking this, `hot_mode_dev_cycle` is the only test I'm aware of where we need a pinned physical device and pinned host hardware since the benchmark is measuring hot reload times, so time for the host to prep it and the device to apply it.

The `__compile` ones are hostonly benchmarks, they are gathering metrics for how long compilation/subsequent complication takes.  So they do not run on physical mobile devices, but should run on physical hosts pinned to consistent hardware.  Like I don't know why we have a `Mac_pixel_7pro hello_world_android__compile` and `Mac_arm64_android hello_world_android__compile` tests, they are doing the same and should be on `Mac_benchmark`.  

cc @zanderso and @reidbaker I need another set of eyes on this, adding and removing all these bot properties is going to impact the Skiaperf series.

Reply:
Actually @dnfield might be the right person to ping these days, might be time to actually audit all of these benchmarks to see what's really valuable and for someone to own the collection properly.  It seems like a bunch of them are copied in ways that don't make sense.